preprocessing/text_data/04_preprocessing_text_data_v3.ipynb

```diff
@@ -0,0 +1,1955 @@
+{
```

Comment:
<p>Add link for &lt;link to 012_text_data_ingestion_s3&gt; </p><p> </p><p>In this sentence: </p><blockquote>but you do not need to run through every single step above.</blockquote><p>Should is say "below" rather than "above"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/1602/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>